syftlib/publish.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: 'com.novoda.bintray-release'
+
+publish {
+
+    def groupProjectID = 'org.openmined'
+    def artifactProjectID = 'syftlib'
+    def publishVersionID = '0.0.1'
```

Comment:
Should we go for automatic versioning probably using git tags? I think that will be better. Something along the lines of nebula-release

Reply:
That'll be nice. If you think the rest is good, we could merge this as it is and add the improvement in a separate PR.

Reply:
(maybe even change the plugin and use nebula)